### PR TITLE
Skip cmethods for 3cities

### PR DESCRIPTION
**Pull request by andrewphilipsmith**

### CODE REVIEW

File: .gitignore
Comment (griff-rees) [inline .gitignore:1]: Fine to add these.

File: compose.yml
Comment (griff-rees) [inline compose.yml:1]: There are two files in `/compose/` meant to provide this functionality
- `/compose/linux-compose.yml`
- `/compose/mac-compose.yml`
See a new ticket bullet point on improving documentation: https://github.com/alan-turing-institute/clim-recal/issues/150

File: python/clim_recal/debiasing/preprocess_data.py
Comment (griff-rees) [inline python/clim_recal/debiasing/preprocess_data.py:1]: See `python/clim_recal/debiasing/preprocess_data.py` in https://github.com/alan-turing-institute/clim-recal/pull/145/files. This should address two ways in which this module is imported.

**Comment by andrewphilipsmith** [comment]
The main purpose of this branch and PR was to share the process of getting the server up and running. Any changes to the code are purely to support that purpose and are superseded by other changes or issues.

I will close this branch/PR unmerged.

**Pull request closed without merging**